notify/discord/discord.go

```diff
@@ -93,34 +128,147 @@ func (n *Notifier) Notify(ctx context.Context, as ...*types.Alert) (bool, error)
 
 	level.Debug(n.logger).Log("incident", key)
 
-	alerts := types.Alerts(as...)
 	data := notify.GetTemplateData(ctx, n.tmpl, as, n.logger)
 	tmpl := notify.TmplText(n.tmpl, data, &err)
 	if err != nil {
 		return false, err
 	}
 
-	title, truncated := notify.TruncateInRunes(tmpl(n.conf.Title), maxTitleLenRunes)
+	author, truncated := notify.TruncateInRunes(tmpl(n.conf.BotUsername), maxEmbedAuthorNameLenRunes)
 	if err != nil {
 		return false, err
 	}
 	if truncated {
-		level.Warn(n.logger).Log("msg", "Truncated title", "key", key, "max_runes", maxTitleLenRunes)
+		level.Warn(n.logger).Log("msg", "Truncated author name", "key", key, "max_runes", maxEmbedAuthorNameLenRunes)
 	}
-	description, truncated := notify.TruncateInRunes(tmpl(n.conf.Message), maxDescriptionLenRunes)
+
+	alertsOmittedMessage, truncated := notify.TruncateInRunes(tmpl(n.conf.AlertsOmittedMessage), maxMessageContentLength)
 	if err != nil {
 		return false, err
 	}
 	if truncated {
-		level.Warn(n.logger).Log("msg", "Truncated message", "key", key, "max_runes", maxDescriptionLenRunes)
+		level.Warn(n.logger).Log("msg", "Truncated alerts omitted message", "key", key, "max_message_length", maxMessageContentLength)
 	}
 
-	color := colorGrey
-	if alerts.Status() == model.AlertFiring {
-		color = colorRed
+	w := webhook{
+		Username:  author,
+		AvatarURL: tmpl(n.conf.BotIconURL),
+	}
+	if err != nil {
+		return false, err
 	}
-	if alerts.Status() == model.AlertResolved {
-		color = colorGreen
+
+	alerts := types.Alerts(as...)
+
+	for alertIndex, alert := range alerts {
```

Comment:
Can use idiomatic variable name `i` here:

```suggestion
	for i, alert := range alerts {
```